### DISCUSSION TITLE
Color themes

### DISCUSSION THREAD

**Discussion by ChefDaddyJay** [jdwilkin4/fcc-forum-clone#52] [Ideas]
The fCC forum has a few different color theme options to choose from. We could clone theirs and/or create our own. Just make sure everything has a good enough contrast ratio. (If only we had a tool for that...)

**Comment by jdwilkin4** [comment]
I am not a huge fan of fcc's dawn theme. For me it is to bright. I vote to pick a light-theme color that is not as bright. 

For the actual theme design were you thinking of a basic dropdown of the different themes in the navbar? 
Something similar to the actual forum?


<img width="325" alt="Screen Shot 2022-11-18 at 5 29 53 PM" src="https://user-images.githubusercontent.com/67210629/202827633-230b2e21-bed8-4ceb-a49f-6e0485e88017.png">

**Reply by ChefDaddyJay** [reply]
Yeah, I was picturing the same three-bar menu in the right corner, and maybe add a link in the left corner to the fCC curriculum to keep the navbar balanced. I'm a fan of symmetry.

![forum-clone-filter-example-4](https://user-images.githubusercontent.com/97722492/202914896-c7f0f860-8489-44a3-9977-69ccae0e76b2.jpg)
